cms/templates/js/container-unit-access.underscore

```diff
@@ -0,0 +1,15 @@
+<%
+var selectedGroupsLabel = userPartitionInfo['selected_groups_label'];
+var selectedPartitionIndex = userPartitionInfo['selected_partition_index'];
+%>
+<% if (selectedGroupsLabel) { %>
+    <small class="access-message"><%-
```

Comment:
For this I was referencing the HTML of the Section/Subsection navigation links in edx-platform/cms/templates/container.html (HTML class "navigation navigation-parents subtitle"), but it's no problem to switch to styling with CSS if we want to do that